### PR TITLE
[AAP-13795] Add Activation missing last restarted field

### PR DESCRIPTION
**Pull request by Dostonbek1**
- Add missing last restarted date field on Activations Read Endpoint: `http://localhost:8000/api/eda/v1/activations/<activation_id>/`

### CODE REVIEW

File: tests/integration/api/test_activation.py [hunk 1]
```diff
@@ -334,6 +335,15 @@ def test_retrieve_activation(client: APIClient, with_project):
     assert data["extra_var"] == {
         "id": activation.extra_var.id,
     }
+    activation_instances = models.ActivationInstance.objects.filter(
+        activation_id=activation.id
+    )
+    if activation_instances:
```
[the remark this answers is not in the document]
Comment (Alex-Izquierdo) [reply tests/integration/api/test_activation.py:341]: Ok, as we discussed, it seems a more general problem of these tests that would require a refactor.

**Review by Alex-Izquierdo — APPROVED** [review]
LGTM, it works as expected.

**Review by hsong-rh — APPROVED** [review]

File: src/aap_eda/api/views/activation.py [hunk 2]
```diff
@@ -374,6 +374,11 @@ def _get_activation_dependent_objects(self, activation):
             activation_id=activation["id"]
         )
         activation["instances"] = activation_instances
+        activation["restarted_at"] = (
```
Comment (Alex-Izquierdo) [inline src/aap_eda/api/views/activation.py:377]: Hi @Dostonbek1 I think this logic is wrong. you are setting the field from the first instance, but if there is only one instance, the activation has not been restarted so this field should not have value

Comment (Alex-Izquierdo) [reply src/aap_eda/api/views/activation.py:377]: Also, the restart count is only increased with automatic restarts, so returning a value when restart count is 0 (the activation has been disabled/enabled manually but it has more than one instance) would be confusing.

Comment (Alex-Izquierdo) [reply src/aap_eda/api/views/activation.py:377]: https://github.com/ansible/eda-server/pull/385

**Pull request merged**